src/sidebar/test/store-test.js

```diff
@@ -190,4 +190,56 @@ describe('store', function () {
       });
     $httpBackend.flush();
   });
+
+  context('when an API calls fail', function () {
+    util.unroll('rejects the call with an Error', function (done, testCase) {
```

Comment:
How about `parametrize()`? Might as well be consistent with the Python code when we can